Branch-SDK/src/main/java/io/branch/referral/ServerResponse.java

```diff
@@ -106,6 +112,14 @@ public JSONArray getArray() {
         return null;
     }
 
+    /**
+     * Set the passed request id
+     *
+     * @param requestId_ passed in from the response headers
+     */
+    public void setRequestId_(@Nullable String requestId_) {
```

Comment:
Let's get rid of this, see the comment on `processEntityForJSON` for more info

Reply:
I created this to keep usages in the future if someone want to pull thisi id apart from the logs, but I'm fine to remove it